src/main/java/eod/card/concrete/normal/SupportAttack.java

```diff
@@ -25,8 +28,11 @@ public SupportAttack() {
 
     @Override
     public void applyEffect(EffectExecutor executor) {
-        Character c = (Character) player.selectObject(WarObject(player.getBoard()).which(Being(Character.class)).which(OwnedBy(player)).get());
-        player.registerListener(new EnemyAttack(c));
+        new EnemyAttack((Character) player.selectObject(
```

Comment:
Same reason as [this](https://github.com/Elegy-Composer/EOD-DM/pull/35/files#r430119252)